### ISSUE TITLE
Redeem For Beneficiary Missing

### ISSUE THREAD

**Issue by dOrgJelli** [daostack/alchemy#1109]
In the context of this proposal: https://alchemy.daostack.io/dao/0x294f999356ed03347c7a23bcbcf8d33fa41dc830/proposal/0x0fccbf2b1374f370d03ccc06054225b644dafd60a00a55be692bd3e222552c8c

As the proposer, I was able to redeem this passed proposal using the "Redemptions" view, which is how I redeemed it for the beneficiary. If you view the proposal directly, there is no option to redeem for the beneficiary as it was taken out of Alchemy recently (if I'm not mistaken).

The problem is, when redeeming this proposal, part of the reward was not executed (the 1K DAI) because the DAO lacks the funds. Now as a user of Alchemy, I can no longer invoke the redemption function again since I have redeemed my reward as the proposer.

Long story short: bring back redeem for beneficiary button pls?

**Comment by jellegerbrandy** [comment]
should be fixed :-)
